tests/integration/cli/test_plugins.py

```diff
@@ -11,3 +11,11 @@ def test_list_excludes_core_plugins(ape_cli, runner):
     assert "console" not in result.output, "console is not supposed to be in Installed Plugins"
     assert "networks" not in result.output, "networks is not supposed to be in Installed Plugins"
     assert "geth" not in result.output, "networks is not supposed to be in Installed Plugins"
+
+
+@pytest.mark.xfail(strict=False, reason="Requires plugins installed")
```

Comment:
if you want to run tests in a separate CI environment, or perhaps after another intermediate step, you can use markers to acheive this: https://docs.pytest.org/en/6.2.x/example/markers.html

so basically, do like:

```
pytest -m "no:plugins_installed" ...
ape install plugin-name
pytest -m "plugins_installed" ...
```

And then have `no:plugins_installed" be the default config for pytest

Reply:
okay ill take a look into this thank you